adoc/admin-security-rbac.adoc

```diff
@@ -93,7 +93,7 @@ kubectl create rolebinding admin --clusterrole=admin --user=<USER_1> --user=<USE
 Administrators can update the authentication connector settings after {productname}
 deployment as follows:
 
-. Edit and save Dex addon YAML in `~/clusters/<CLUSTER_NAME>/addons/dex/dex.yml`
+. Copy Dex base configuration `~/clusters/<CLUSTER_NAME>/addons/dex/base/dex.yml` to `~/clusters/<CLUSTER_NAME>/addons/dex/patches/custom.yml` then edit and save Dex custom settings addon YAML in `~/clusters/<CLUSTER_NAME>/addons/dex/patches/custom.yml`
```

Comment:
On the patches directory they should provide a kustomize patch of the form of [strategic merge patch](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#patchstrategicmerge) or a [JSON 6902 patch](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#patchjson6902).

The base manifest is just a way for the user to know how they can patch it, they should not copy it under the patches directory, but write a valid patch for that given base manifest.

Reply:
I fix this comment